services/brig/src/Brig/Data/UserPendingActivation.hs

```diff
@@ -38,7 +38,6 @@ data UserPendingActivation = UserPendingActivation
   }
   deriving stock (Eq, Show, Ord)
 
--- | Note: Call this function only after an invitation for the user has been created
```

Comment:
2a64d2825a101c181907417b2df6374035ef7834

Reply:
ah, nice. Didn't see that.